model/clusters_mgmt/v1/cluster_type.model

```diff
@@ -261,4 +261,8 @@ class Cluster {
 	// The AutoNode settings for this cluster.
 	// This is currently only supported for ROSA HCP
 	AutoNode ClusterAutoNode
+
+	// Log forwarders configuration for the cluster.
+	// This can be set during cluster creation to configure log forwarders.
+	LogForwarders []LogForwarder
```

Comment:
Other fields seem to have `link` before the array e.g see `ingresses`, `addons` . Can you please add it.